<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BLRM-300068692" failures="0" tests="1" name="com.qapitol.automationqa.apitest.serviceTest.TestService" time="1.584" errors="1" timestamp="13 Dec 2019 05:31:19 GMT" skipped="0">
  <testcase classname="com.qapitol.automationqa.apitest.serviceTest.TestService" name="CheckImage" time="1.584">
    <error message="Unrecognized field &quot;_id&quot; (class com.qapitol.automationqa.api.pojo.Me), not marked as ignorable (12 known properties: &quot;statusConnection&quot;, &quot;status&quot;, &quot;settings&quot;, &quot;active&quot;, &quot;emails&quot;, &quot;name&quot;, &quot;roles&quot;, &quot;services&quot;, &quot;utcOffset&quot;, &quot;id&quot;, &quot;username&quot;, &quot;avatarUrl&quot;])
 at [Source: (String)&quot;{&quot;status&quot;:&quot;success&quot;,&quot;data&quot;:{&quot;userId&quot;:&quot;DZrpycKxidxuhtPdz&quot;,&quot;authToken&quot;:&quot;7QUyCu5SwgEIibwVadxHJv58avg0GZOoBM-vna9RGP3&quot;,&quot;me&quot;:{&quot;_id&quot;:&quot;DZrpycKxidxuhtPdz&quot;,&quot;services&quot;:{&quot;password&quot;:{&quot;bcrypt&quot;:&quot;$2b$10$7VbLaiW9dezYbky0kTM1K.p6LVdQkL9agutywFu4qe.pEm.6JNptS&quot;}},&quot;emails&quot;:[{&quot;address&quot;:&quot;bairagi1007@live.com&quot;,&quot;verified&quot;:false}],&quot;status&quot;:&quot;online&quot;,&quot;active&quot;:true,&quot;roles&quot;:[&quot;user&quot;],&quot;name&quot;:&quot;qapios&quot;,&quot;statusConnection&quot;:&quot;online&quot;,&quot;utcOffset&quot;:5.5,&quot;username&quot;:&quot;qapios&quot;,&quot;settings&quot;:{&quot;preferences&quot;:{&quot;enableAutoAway&quot;:true,&quot;idleTimeLimit&quot;[truncated 800 chars]; line: 1, column: 129] (through reference chain: com.qapitol.automationqa.api.pojo.UserResponse[&quot;data&quot;]-&gt;com.qapitol.automationqa.api.pojo.Data[&quot;me&quot;]-&gt;com.qapitol.automationqa.api.pojo.Me[&quot;_id&quot;])" type="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException">
      <![CDATA[com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "_id" (class com.qapitol.automationqa.api.pojo.Me), not marked as ignorable (12 known properties: "statusConnection", "status", "settings", "active", "emails", "name", "roles", "services", "utcOffset", "id", "username", "avatarUrl"])
 at [Source: (String)"{"status":"success","data":{"userId":"DZrpycKxidxuhtPdz","authToken":"7QUyCu5SwgEIibwVadxHJv58avg0GZOoBM-vna9RGP3","me":{"_id":"DZrpycKxidxuhtPdz","services":{"password":{"bcrypt":"$2b$10$7VbLaiW9dezYbky0kTM1K.p6LVdQkL9agutywFu4qe.pEm.6JNptS"}},"emails":[{"address":"bairagi1007@live.com","verified":false}],"status":"online","active":true,"roles":["user"],"name":"qapios","statusConnection":"online","utcOffset":5.5,"username":"qapios","settings":{"preferences":{"enableAutoAway":true,"idleTimeLimit"[truncated 800 chars]; line: 1, column: 129] (through reference chain: com.qapitol.automationqa.api.pojo.UserResponse["data"]->com.qapitol.automationqa.api.pojo.Data["me"]->com.qapitol.automationqa.api.pojo.Me["_id"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:840)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:1192)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1592)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1570)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:294)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4202)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3205)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3173)
at com.qapitol.automationqa.api.client.ApiUtils.convertJsonStringToPojoObject(ApiUtils.java:100)
at com.qapitol.automationqa.api.services.client.ServiceClient.doLogin(ServiceClient.java:33)
at com.qapitol.automationqa.apitest.serviceTest.TestService.CheckImage(TestService.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- CheckImage -->
</testsuite> <!-- com.qapitol.automationqa.apitest.serviceTest.TestService -->
